### ISSUE TITLE
TC: Create projects in Tamr

### ISSUE THREAD

**Issue by abafzal** [Datatamer/tamr-client#433]
# 🙋 feature request

Tamr Client Beta should support creation of projects in Tamr

## Examples

```python
tc.project.create
(
    name='name of my project', 
    description='description of my project', 
    type='mastering|categorization|schema mapping|golden records'
)
```

**Comment by ianbakst** [comment]
Thinking about other possible parameters:
`enable_transformations = TRUE|false`

**Comment by pcattori** [comment]
Rather than specifying type, we should just have a `create` function accessed via project type namespace:

```python
tc.mastering.project.create(...)`
```

**Comment by abafzal** [comment]
Would this following be fine with folks: 

```
tc.mastering.project.create(...)
tc.categorization.project.create(...)
tc.schema_mapping.project.create(...)
tc.golden_records.project.create(...)
```

**Comment by pcattori** [comment]
Resolved by #435
